tests/cypress/e2e/languageSettingsUI.cy.ts

```diff
@@ -20,6 +20,8 @@ describe('Tests on UI for language settings', () => {
         LanguageSettings.visit('languagesTestSite', 'ca')
         languageSettings.switchReplaceUntranslatedWithDefault()
         languageSettings.submitChanges()
+        //eslint-disable-next-line
+        cy.wait(5000)
```

Comment:
Please avoid to cy.wait. 
In your case, you should be able to use a waitUntil on the getNodeByPath
```
cy.waitUntil(() =>
            getNodeByPath('/sites/languagesTestSite', ['j:mixLanguage']).then((result) => result.data.jcr.nodeByPath.properties[0].value === true),
```